build.gradle

```diff
@@ -26,6 +26,9 @@ dependencies {
     implementation 'com.github.weisj:darklaf-core:2.2.0'
     implementation 'com.github.weisj:darklaf-theme:2.2.0'
     implementation 'com.formdev:svgSalamander:1.1.2.1'
+    compile 'org.javatuples:javatuples:1.2'
```

Comment:
Is it worth to bring a new dependency just for pair?

Reply:
Yes I know, but this is a very neat dependance (really small and super useful). I am sure we could reuse it somewhere else. Also I think it would be temporary as Java should implement tuple in coming versions.

Reply:
cool, makes sense.